packages/plugins/text/src/controls/index.tsx

```diff
@@ -20,14 +20,7 @@ export interface ControlProps {
   config: TextPluginConfig
   editor: Editor
   pluginClosure: SlatePluginClosure
-  plugins: {
-    suggestions?: boolean
-    math?: boolean
-    headings?: boolean
-    lists?: boolean
-    colors?: boolean
-    code?: boolean
-  }
+  plugins: TextConfig['plugins']
```

Comment:
hier bin ich mir unsicher warum die types gedoppelt waren (aus meiner sicht)
ping @inyono

Reply:
Ah, yeah, that's the problem. `TextConfig['plugins']` also might be undefined because of `plugins?: ...`. So if you want to share the type, you'd have to create a new type for that and use it in those two places (or alternatively do some weird TS voodoo to get rid of the undefined).

Reply:
so overall it make sense to share the types here? Than I'll create a new shared type.